web/app/Http/Controllers/AppController.php

```diff
@@ -183,4 +189,33 @@ public function postDevModeLaunch(Request $request) {
 
         return redirect('/app');
     }
+
+    public function gradePassback(Request $request){
+          $assignment = Assignment::where('resource_link_dbid', $request->session()->get('lti_resource_link_dbid'))
+         ->firstOrFail();
+            
+         set_time_limit(60);
+         ini_set('memory_limit', '2048M');
+         if(!ini_get('auto_detect_line_endings')){
+            ini_set('auto_detect_line_endings', '1');
+         }
+
+         $assignment_response = AssignmentResponse::where('assignment_id',$assignment->id)
```

Comment:
Name this 'assignment_responses' instead, since this is a list of responses rather than a single one.